cmd/mr_list.go

```diff
@@ -172,6 +185,8 @@ func init() {
 		&mrMilestone, "milestone", "", "list only MRs for the given milestone")
 	listCmd.Flags().BoolVarP(&mrAll, "all", "a", false, "list all MRs on the project")
 	listCmd.Flags().BoolVarP(&mrMine, "mine", "m", false, "list only MRs assigned to me")
+	listCmd.Flags().MarkDeprecated("mine", "please --assignee instead")
```

Comment:
```suggestion
	listCmd.Flags().MarkDeprecated("mine", "please use --assignee instead")
```

Reply:
And one other thing :)  don't use 'please'.  I know it sounds rude not to but it's something that you're not supposed to do.  I'm definitely guilty of writing 'polite' instructions too :)

Reply:
Yea, you're right. I'm going to drop the `please` and add the `use` that I forgot.